elephant/test/test_statistics.py

```diff
@@ -570,6 +570,35 @@ def test_instantaneous_rate_regression_144(self):
         st = neo.SpikeTrain([2.12, 2.13, 2.15] * pq.s, t_stop=10 * pq.s)
         self.assertRaises(ValueError, es.instantaneous_rate, st, 1 * pq.ms)
 
+    # Regression test for #245
+    def test_instantaneous_rate_regression_245(self):
+        # This test makes sue that the correct kernel width is chosen when
+        # selecting 'auto' as kernel
+        spiketrain = neo.SpikeTrain(
+            range(1, 30) * pq.ms, t_start=0*pq.ms, t_stop=30*pq.ms)
+
+        # This is the correct procedure to attain the kernel: first, the result
+        # of sskernel retrieves the kernel bandwidth of an optimal Gaussian
+        # kernel in terms of its standard deviation sigma, then uses this value
+        # directly in the function for creating the Gaussian kernel
+        kernel_width_sigma = es.sskernel(
+            spiketrain.magnitude, tin=None, bootstrap=True)['optw']
```

Comment:
Is `bootstrap=True` necessary?

Reply:
Indeed, it seems this could be left out.